contest/b_nom_de_groupe/sol/b_nom_de_groupe.cpp

```diff
@@ -0,0 +1,49 @@
+// Solution by Louis Sugy
+
+#include <iostream>
+#include <string>
+
+int score(std::string name) {
+  int l = name.size();
+  bool bonus = false;
+  const char *name_array = name.c_str();
+
+  int nv = 0, nc = 0;
+#pragma omp parallel for reduction(+ : nv, nc) schedule(guided)
+  for (int i = 0; i < l; i++) {
+    char c = name_array[i];
+    if (c == 'a' || c == 'e' || c == 'i' || c == 'o' || c == 'u' || c == 'y')
+      nv++;
+    else
+      nc++;
```

Comment:
Did you try to only count vowels, and computing consonants as `nc = l - nv`? This way you will need less memory in your reduction. Maybe that may help with performance?